src/messagingsession/PrefetchOn.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+enum PrefetchOn {
```

Comment:
Could you explain in PR description what is `Prefetch` and how it can be expanded for future values?

Reply:
I think since this is getting exported out from library, should this `PrefetchOn` be more specific keyword to messaging. In future, may be another Prefetch comes for meeting session. Just a point to consider before making this public :)